finetune/llama/README.md

```diff
@@ -0,0 +1,53 @@
+### PyTorch LLAMA2 7B lora apalca finetuning
+
+## Description
+This document is a guide for running LLaMA2 7B lora apalca finetuning using PyTorch on CPU.
+
+## Step-by-step run guide
+# Prepare dependency
+wget https://intel-extension-for-pytorch.s3.amazonaws.com/ipex_stable/cpu/oneccl_bind_pt-2.0.0%2Bcpu-cp39-cp39-linux_x86_64.whl
```

Comment:
it would be better to display the relevant code commands using the markdown code format.
https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

Reply:
You can directly place the WHL package URL into the Python `requirements.txt` file, and pip will automatically download it. However, first, you need to specify the Python version. The package you provided is only compatible with Python `3.9`.

Reply:
> it would be better to display the relevant code commands using the markdown code format. https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

Hi, thx for the comments. Updated accordingly.